### PR TITLE
fix: serialize SVGs correctly

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Serializes SVGs as a complete XML document, meaning that nodes aren't dropped. (This includes processing instructions, which might reference a stylesheet.)

### CODE REVIEW

**Comment by unindented** [comment]
@thewilkybarkid can you give a bit more context on how you were using this, and what problem you encountered? I didn't think anyone was using this...

**Comment by thewilkybarkid** [comment]
😅  Have been wanting to try out RoughJS on some existing SVGs, there are a few options (none of which seem to be used much) but this has a simple string-in string-out approach which made it easier.

My SVGs come from Graphviz, and I've been using the `stylesheet` option there which is turned into a processing instruction, but Rougher removed it.

File: src/node/index.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  const { window } = new JSDOM();
+  const { window } = new JSDOM(input, { contentType: "image/svg+xml" });
 
-  window.document.body.innerHTML = input;
   const svg = window.document.querySelector("svg") as SVGSVGElement;
 
   roughUp(svg, options);
 
-  return svg.outerHTML;
+  return serializer(window.document);
```
Comment (unindented) [inline src/node/index.ts:14]: _question_: Why `serializer(window.document)` vs `serializer(svg.outerHTML)`?

Comment (thewilkybarkid) [reply src/node/index.ts:14]: `outerHTML` returns the elements and its descendants, but not its siblings (doctype, comments, whitespace etc).

There is a subtle difference in that previously it returned just the SVG if happened to be embedded (in HTML or whatever), now it returns the whole document.

**Comment by unindented** [comment]
Umm, we got an error when building with Rollup: https://github.com/unindented/rougher/runs/3198643733?check_suite_focus=true

```
src/node/index.ts → dist/node/rougher.mjs...
[!] Error: 'default' is not exported by node_modules/w3c-xmlserializer/lib/serialize.js, imported by src/node/index.ts
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
src/node/index.ts (2:7)
1: import { JSDOM } from "jsdom";
2: import serializer from "w3c-xmlserializer";
```

@thewilkybarkid do you have time to look into it? It may take me a few days.

**Comment by thewilkybarkid** [comment]
> My SVGs come from Graphviz, and I've been using the `stylesheet` option there which is turned into a processing instruction, but Rougher removed it.

Just to add a bit more detail (now I can), I've used it in [parcel-transformer-rough-svg](https://github.com/thewilkybarkid/parcel-transformer-rough-svg) which I'm using in combination with [parcel-transformer-graphviz](https://github.com/thewilkybarkid/parcel-transformer-graphviz). (My usage isn't public yet, but I'm referencing Graphviz files from HTML which Parcel turns into roughened SVGs.)

**Pull request merged**